tests/end_to_end_tests/run_in_remote_machine/cleanup_test_ec2_objects.py

```diff
@@ -0,0 +1,225 @@
+import argparse
```

Comment:
IIRC, there is duplicated between this and existing AMI clean up script, right?

I assume that eventually we will be able to get rid all of the old / duplicated code?